src/io/jenkins/infra/InfraConfig.groovy

```diff
@@ -27,4 +41,38 @@ class InfraConfig implements Serializable {
       return 'jenkins4eval'
     }
   }
+
+  // Returns the Docker Informations for pulling images
+  def getDockerPullOrgAndCredentialsId() {
+    switch(jenkinsHostname){
+      case "ci.jenkins.io": 
+        return [error: false, organisation: "cijenkinsioinfra", credentialId: "cijenkinsioinfra-dockerhub-pull"]
+        break;
+      case "trusted.ci.jenkins.io": 
+        return [error: false, organisation: "trustedcijenkinsio", credentialId: "trustedcijenkinsio-dockerhub-pull"]
+        break;
+      case "infra.ci.jenkins.io": 
+        return [error: false, organisation: "infracijenkinsio", credentialId: "infracijenkinsio-dockerhub-pull"]
+        break;
+      default:
+        return [error: true, msg: "Cannot use Docker credentials outside of jenkins infra environments"]
+    }
+  }
+
+  // Returns the Docker Informations for pushing images
+  def getDockerPushOrgAndCredentialsId() {
+    switch(jenkinsHostname){
+      case "ci.jenkins.io": 
+        return [error: false, organisation: "jenkins4eval", credentialId: "jenkins4eval-dockerhub-push"]
+        break;
+      case "trusted.ci.jenkins.io": 
+        return [error: false, organisation: "jenkins", credentialId: "jenkins-dockerhub-push"]
+        break;
+      case "infra.ci.jenkins.io": 
+        return [error: false, organisation: "jenkinsciinfra", credentialId: "jenkinsciinfra-dockerhub-push"]
```

Comment:
```suggestion
        return [error: false, organisation: "jenkinsinfraadmin", credentialId: "jenkinsciinfra-dockerhub-push"]
```

As we discovered together, the account `jenkinsinfraadmin` is the owner of the organization `jenkinsciinfra` so we have to keep it.